lib/install.js

```diff
@@ -31,34 +34,29 @@ function install (gyp, argv, callback) {
     if (err) {
       log.warn('install', 'got an error, rolling back install')
       // roll-back the install if anything went wrong
-      gyp.commands.remove([ version ], function (err2) {
+      gyp.commands.remove([ release.versionStr ], function (err2) {
```

Comment:
This should't be `versionDir`, right? i can't tell where `version` came from here..